README.md

```diff
@@ -1,80 +1,67 @@
 
-## Running the Samples
-To run the samples in a terminal, `cd` into the project and call the gradle `run` task, supplying the sample main class:
+### IntelliJ IDEA
 
-On Linux/Mac
-```
-$ ./gradlew run -PmainClass="com.esri.samples.scene.display_scene.DisplaySceneSample"
-```
+1. Open IntelliJ IDEA and select _File > Open..._.
+2. Choose one of the sample project directories (not the category folder) and click _OK_.
+3. Select _File > Project Structure..._ and ensure that the Project SDK and language level are set to use Java 11.
+4. Open the Gradle view with _View > Tool Windows > Gradle_.
+5. In the Gradle view, double-click the `run` task under _Tasks > application_ to run the app.
 
-On Windows
-```
-> gradlew.bat run -PmainClass="com.esri.samples.scene.display_scene.DisplaySceneSample"
-```
+### Eclipse
 
-If no `mainClass` property is supplied, the default sample (set in the build.gradle file) will be run. 
+1. Open Eclipse and select _File > Import_.
+2. In the import wizard, choose _Gradle > Existing Gradle Project_, then click _Next_.
+3. Choose one of the sample project directories (not the category folder) as the project root directory.
+4. Click _Finish_ to complete the import.
+5. Open the Gradle Tasks view with _Window > Show View > Other... > Gradle > Gradle Tasks_.
+6. In the Gradle Tasks view, double-click the `run` task under _{project_name} > application_ to run the app.
 
-There is no need to install Gradle to run the samples.
+### Terminal
 
-## Java 11
+1. `cd` into one of the sample project directories (not the category folder).
+2. Run `./gradlew run` on Linux/Mac or `gradlew.bat run` on Windows to run the app.
+
+### Java 11
 Java 11 users may find exceptions when running the project if their library path is still set for Oracle JDK 1.8 (see the [OpenJavaFX docs](https://openjfx.io/openjfx-docs/) for more information). A workaround for this is to add the following argument in the `run` task of the Gradle buildscript:
 ```
 systemProperty "java.library.path", "C:\tmp"
 ```
 
-## Offline sample data
+### Offline sample data
 Some samples require offline data. A `samples-data` directory will automatically download to the project root the 
-first time you call the `run` task.
-
-## Importing into an IDE
-We will step through how to import the Samples project into Eclipse and IntelliJ IDEA. Other IDEs may support Gradle too. Please consult their documentation for importing Gradle projects.
-
-### IntelliJ IDEA
-After cloning the samples, open IntelliJ IDEA and follow these steps:
-
-1. Click *Import Projects* from the Welcome Screen or select *File > New > Project from Existing Sources*.
-2. In the select path dialog, select the `build.gradle` file in the `arcgis-runtime-samples-java` directory. Click *OK* after specifying the `build.gradle` file.
-3. Click *OK* at the next dialog to complete the import.
+first time you call the `run` task. To refresh the data, you can run the `clean` and `unpackData` tasks.
```

Comment:
minor thing, looks like the continuation of this sentence has jumped onto a new line